src/lib/Zikula/Bundle/CoreBundle/Twig/Extension/CoreExtension.php

```diff
@@ -365,6 +366,20 @@ public function getModVar($module, $name, $default = null)
     }
 
     /**
+     * @param $name
+     * @param null $default
+     * @return mixed
+     */
+    public function getSystemVar($name, $default = null)
+    {
+        if (empty($name)) {
+            throw new \InvalidArgumentException(__('Empty argument at') . ':' . __FILE__ . '::' . __LINE__);
```

Comment:
Why does the exception contain the file name and line? This is redundant information.